components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Core/Models/SimpleAttributeOperandModel.cs

```diff
@@ -29,5 +29,10 @@ public class SimpleAttributeOperandModel {
         /// Index range of attribute operand
         /// </summary>
         public string IndexRange { get; set; }
+
+        /// <summary>
+        /// Optional display name
+        /// </summary>
+        public string DisplayName { get; set; }
```

Comment:
Do we really need 'DisplayName'? It's never used and SimpleAttributeOperand in OpcUA spec does not contain a DisplayName field